lua/telescope/make_entry.lua

```diff
@@ -240,6 +240,38 @@ do
   end
 end
 
+function make_entry.gen_from_git_stash()
+  local displayer = entry_display.create {
+    separator = " ",
+    items = {
+      { width = 50 },
+      { remaining = true }
+      }
+    }
+
+  local make_display = function(entry)
+    return displayer {
+      -- {entry.value, "TelescopeResultsIdentifier"},
+      entry.msg
```

Comment:
you have two item columns and you only push in one?

If you do it like this. display can only be `display = splitted[2]`

Reply:
I fixed it, thank you, I am not familiar with telescope internal terminology and I used code I think from git commits for this.

Reply:
no worries :)